src/nifreeze/cli/parser.py

```diff
@@ -164,3 +164,106 @@ def build_parser() -> ArgumentParser:
     )
 
     return parser
+
+
+def _determine_single_fit_mode(model_name: str) -> bool:
+    """Determine if a model is to be run in *single-fit mode*.
+
+    If a model is requested to be run in *single-fit mode*, it will be run only
+    once in all the data available.
+
+    Parameters
+    ----------
+    model_name : :obj:`str
+        Model name.
+
+    Returns
+    -------
+    :obj:`bool`
+        ``True`` if the model is to be run in *single-fit mode*, ``False``
+        otherwise.
+    """
+
+    return model_name.lower().startswith("single")
+
+
+def _normalize_model_name(model_name: str) -> str:
+    """Normalize a model name.
+
+    Normalize a model name by converting it to all lowercase and stripping the
+    ``single`` prefix if present.
+
+    Parameters
+    ----------
+    model_name : :obj:`str`
+        Model name.
+
+    Returns
+    -------
+    :obj:`str`
+        Normalized model name.
+    """
+
+    return model_name.lower().replace("single", "")
```

Comment:
IMHO, spawning one-liner string manipulations into functions is less transparent

Reply:
OK, now I see another reason why that went into a separate function:
https://github.com/nipreps/nifreeze/actions/runs/16146097132/job/45565241798?pr=106#step:11:671

Are we OK if I bring them back, and have this PR merged? I do not want to go over more failures...